rlkit/demos/spacemouse/input_server.py

```diff
@@ -0,0 +1,116 @@
+# first to start the nameserver start: python -m Pyro4.naming
+
+import Pyro4
+from threading import Thread
+import time
+import numpy as np
+from rlkit.launchers import config
+# HOSTNAME = "192.168.0.102"
+
+Pyro4.config.SERIALIZERS_ACCEPTED = set(['pickle','json', 'marshal', 'serpent'])
+Pyro4.config.SERIALIZER='pickle'
+
+device_state = None
+
+@Pyro4.expose
+class DeviceState(object):
+    state = None
+
+    def get_state(self):
+        return device_state
+        # return self.state
+
+    def set_state(self, state):
+        # print("set", state)
+        # self.state = state
```

Comment:
it'd be nice to remove some of these commented out code lines (also in other files)

Reply:
Done, and went through and cleaned out more comments and unused files I recognized